examples/tutorial/tutorial_notebooks/introduction/pandas_on_ray/exercise_1.ipynb

```diff
@@ -115,7 +115,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
```

Comment:
Let's do the stuff related to pandas_on_ray as part of a separate issue.

Reply:
@YarShev this was changed

Reply:
We should be done as part of a separate issue.